clang/include/clang/AST/TypeLoc.h

```diff
@@ -942,6 +942,7 @@ class BTFTagAttributedTypeLoc
 
 struct HLSLAttributedResourceLocInfo {
   SourceRange Range;
+  TypeSourceInfo *ContainedTyTSI;
```

Comment:
`ContainedTyInfo` might be a better name - we usually only use type abbreviations if its the whole variable name.